pkg/querier/seriesset.go

```diff
@@ -100,7 +102,7 @@ func (s *V3ioSeriesSet) Next() bool {
 
 		s.nullSeries = false
 
-		if s.aggrSeries.CanAggregate(s.partition.AggrType()) && s.maxt-s.mint >= s.interval {
+		if s.canAggregate {
```

Comment:
you sure you want to delete this condition ` && s.maxt-s.mint >= s.interval`? why isn't it relevant anymore?

Reply:
@imakhlin not relevant (old condition)